Rethinking_2/Chp_07.ipynb

```diff
@@ -0,0 +1,3517 @@
+{
```

Comment:
<p>instead of waic use the attribute waic_i (I will make this more clear in the arviz documentation). You can also set the scale (deviance, log or negative_log) and default method (waic, loo) using rcParams ('stats.ic_scale' and 'stats.information_criterion')</p><p> </p><p>You should do something like</p><p> </p><p>waic_m_6_7 = az.waic(m_6_7_trace, pointwise=True, scale='deviance')</p><p>waic_m_6_8 = az.waic(m_6_8_trace, pointwise=True, scale='deviance')</p><p> </p><p>diff_m6.7_m6.8 = waic_m_6_7.waic_i - waic_m_6_8.waic_i</p><p>n = len(diff_m6.7_m6.8)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='68'/>